integrations/gitlab/gitlab_integration/models/webhook_config_models.py

```diff
@@ -0,0 +1,19 @@
+from pydantic import BaseModel
+
+
+class WebhookGroup(BaseModel):
+    events: list[str]
+
+
+class WebhookToken(BaseModel):
+    groups: dict[str, WebhookGroup]
```

Comment:
the name of the class doesn't represent the actual properties and types inside of it. The object doesn't contain anything related to token

Reply:
done